tools/local-env/scripts/install.js

```diff
@@ -13,7 +13,10 @@ dotenvExpand.expand( dotenv.config() );
 local_env_utils.determine_auth_option();
 
 // Create wp-config.php.
-wp_cli( `config create --dbname=wordpress_develop --dbuser=root --dbpass=password --dbhost=mysql --force --config-file=${process.env.LOCAL_DIR}/../wp-config.php` );
+wp_cli( `config create --dbname=wordpress_develop --dbuser=root --dbpass=password --dbhost=mysql --force --config-file="wp-config.php"` );
+
+// Since WP-CLI runs as root, the wp-config.php created above will be read-only. This needs to be writable for the sake of E2E tests.
+execSync( 'node ./tools/local-env/scripts/docker.js exec cli chmod 666 wp-config.php' );
```

Comment:
It's not clear to me why this is needed now. Without it, E2E tests fail due to `wp-config.php` not being writable: https://github.com/WordPress/wordpress-develop/actions/runs/15627027244/job/44023152411

In 0e4ddbb01872edb06847fcd4c2d290a8ba34cc2e I tried using the node API to set the file permissions, but this failed because it's not being run as root: https://github.com/WordPress/wordpress-develop/actions/runs/15624512371/job/44016093197

Reply:
I think in https://github.com/WordPress/wordpress-develop/commit/0e4ddbb01872edb06847fcd4c2d290a8ba34cc2e it was running on the local filesystem whereas in this implementation it is running in the container.